zerver/views/video_calls.py

```diff
@@ -184,20 +195,23 @@ def get_bigbluebutton_url(request: HttpRequest, user_profile: UserProfile) -> Ht
 @zulip_login_required
 @never_cache
 @has_request_variables
-def join_bigbluebutton(request: HttpRequest, meeting_id: str = REQ(validator=check_string),
-                       password: str = REQ(validator=check_string),
-                       checksum: str = REQ(validator=check_string)) -> HttpResponse:
+def join_bigbluebutton(request: HttpRequest, meeting_id: str = REQ(),
+                       name: str = REQ(), password: str = REQ(),
+                       checksum: str = REQ()) -> HttpResponse:
```

Comment:
Why are we losing the `validator`s here?

Reply:
See andersk comment: https://github.com/zulip/zulip/pull/16580#discussion_r510739171